tests/end2end_test.py

```diff
@@ -59,7 +59,7 @@ def end2end(config_file, ref_dsm, absmean_tol=0.025, percentile_tol=1.):
 
     # TODO: this is ugly, and will be fixed once we'll have implemented a better
     # way to control the config parameters
-    if 'utm_zone' in s2p.cfg: del s2p.cfg['utm_zone']
+    if 'out_crs' in s2p.cfg: del s2p.cfg['out_crs']
```

Comment:
We currently do not test any non-default `out_crs` in the tests, but we might also not want to add an extra end-to-end test just for that, because they take a long time.

@carlodef do you have ideas of how we could meaningfully test `out_crs` without a full-blown end-to-end test?

Reply:
Sounds good, that's what I intended to do!

Reply:
I've added tests in 188890e604597d8be779e730c6539422627dbf64

I didn't have your comment precisely in mind though, so I did tests on the higher-level `s2p.disparity_to_ply`. Is that OK?

Reply:
That's perfect, thanks!